<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/recursos/paleta.png</normaloff>:/recursos/paleta.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEstilo">
    <property name="title">
     <string>Estilo</string>
    </property>
    <addaction name="actionColor"/>
    <addaction name="actionAncho"/>
   </widget>
   <widget class="QMenu" name="menuForma">
    <property name="title">
     <string>Forma</string>
    </property>
    <addaction name="actionLineas"/>
    <addaction name="actionLibre"/>
    <addaction name="actionRect_nculos"/>
    <addaction name="actionCircunferencias"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEstilo"/>
   <addaction name="menuForma"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuevo"/>
   <addaction name="actionGuardar"/>
   <addaction name="actionAbrir"/>
   <addaction name="actionColor"/>
   <addaction name="actionAncho"/>
   <addaction name="actionLibre"/>
   <addaction name="actionLineas"/>
   <addaction name="actionRect_nculos"/>
   <addaction name="actionCircunferencias"/>
   <addaction name="actionSalir"/>
  </widget>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/recursos/registro.png</normaloff>:/recursos/registro.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/recursos/salvar.png</normaloff>:/recursos/salvar.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/recursos/puerta-de-salida.png</normaloff>:/recursos/puerta-de-salida.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/recursos/rueda-de-color.png</normaloff>:/recursos/rueda-de-color.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionAncho">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/recursos/ancho-del-texto.png</normaloff>:/recursos/ancho-del-texto.png</iconset>
   </property>
   <property name="text">
    <string>Ancho</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionLineas">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/recursos/menu.png</normaloff>:/recursos/menu.png</iconset>
   </property>
   <property name="text">
    <string>Lineas</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionLibre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/recursos/libro-de-arte.png</normaloff>:/recursos/libro-de-arte.png</iconset>
   </property>
   <property name="text">
    <string>Libre</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRect_nculos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/recursos/rectangulo.png</normaloff>:/recursos/rectangulo.png</iconset>
   </property>
   <property name="text">
    <string>Rect√°nculos</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCircunferencias">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/recursos/circulo.png</normaloff>:/recursos/circulo.png</iconset>
   </property>
   <property name="text">
    <string>Circunferencias</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/recursos/abrir-documento.png</normaloff>:/recursos/abrir-documento.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
